examples/md-flexible/src/Simulation.cpp

```diff
@@ -372,6 +465,106 @@ void Simulation::calculateGlobalForces(const std::array<double, 3> &globalForce)
   }
 }
 
+void Simulation::logSimulationState() {
+  int totalNumberOfParticles, ownedParticles, haloParticles;
+
+  int particleCount = _autoPasContainer->getNumberOfParticles(autopas::IteratorBehavior::ownedOrHalo);
+  autopas::AutoPas_MPI_Allreduce(&particleCount, &totalNumberOfParticles, 1, AUTOPAS_MPI_INT, AUTOPAS_MPI_SUM,
+                                 AUTOPAS_MPI_COMM_WORLD);
+
+  particleCount = _autoPasContainer->getNumberOfParticles();
+  autopas::AutoPas_MPI_Allreduce(&particleCount, &ownedParticles, 1, AUTOPAS_MPI_INT, AUTOPAS_MPI_SUM,
+                                 AUTOPAS_MPI_COMM_WORLD);
+
+  particleCount = _autoPasContainer->getNumberOfParticles(autopas::IteratorBehavior::halo);
+  autopas::AutoPas_MPI_Allreduce(&particleCount, &haloParticles, 1, AUTOPAS_MPI_INT, AUTOPAS_MPI_SUM,
+                                 AUTOPAS_MPI_COMM_WORLD);
+
+  double standardDeviationOfHomogeneity;
+  autopas::AutoPas_MPI_Allreduce(&_homogeneity, &standardDeviationOfHomogeneity, 1, AUTOPAS_MPI_DOUBLE, AUTOPAS_MPI_SUM,
+                                 AUTOPAS_MPI_COMM_WORLD);
+  standardDeviationOfHomogeneity = standardDeviationOfHomogeneity / _domainDecomposition.getNumberOfSubdomains();
+
+  std::cout << "\n\n"
+            << "Total number of particles at the end of Simulation: " << totalNumberOfParticles << "\n"
+            << "Owned: " << ownedParticles << "\n"
+            << "Halo: " << haloParticles << "\n"
+            << "Standard Deviation of Homogeneity: " << standardDeviationOfHomogeneity << std::endl;
+}
+
+void Simulation::logMeasurements() {
+  long positionUpdate = accumulateTime(_timers.positionUpdate.getTotalTime());
+  long forceUpdateTotal = accumulateTime(_timers.forceUpdateTotal.getTotalTime());
+  long forceUpdatePairwise = accumulateTime(_timers.forceUpdatePairwise.getTotalTime());
+  long forceUpdateGlobalForces = accumulateTime(_timers.forceUpdateGlobal.getTotalTime());
+  long forceUpdateTuning = accumulateTime(_timers.forceUpdateTuning.getTotalTime());
+  long forceUpdateNonTuning = accumulateTime(_timers.forceUpdateNonTuning.getTotalTime());
+  long velocityUpdate = accumulateTime(_timers.velocityUpdate.getTotalTime());
+  long simulate = accumulateTime(_timers.simulate.getTotalTime());
+  long vtk = accumulateTime(_timers.vtk.getTotalTime());
+  long initialization = accumulateTime(_timers.initialization.getTotalTime());
+  long total = accumulateTime(_timers.total.getTotalTime());
+  long thermostat = accumulateTime(_timers.thermostat.getTotalTime());
+  long haloParticleExchange = accumulateTime(_timers.haloParticleExchange.getTotalTime());
+  long migratingParticleExchange = accumulateTime(_timers.migratingParticleExchange.getTotalTime());
+
+  if (_domainDecomposition.getDomainIndex() == 0) {
+    auto maximumNumberOfDigits = std::to_string(total).length();
+    std::cout << "Measurements:" << std::endl;
+    std::cout << timerToString("Total                        ", total, maximumNumberOfDigits, total);
```

Comment:
```suggestion
    std::cout << timerToString("Total accumulated            ", total, maximumNumberOfDigits);
```
If you reintroduce the default arguments of this function you could do this.

Also maybe hint to the fact that this is accumulated time.